app/posts/manage-teacher-training-applications/2023-03-13-revisiting-reasons-for-rejection.md

```diff
@@ -0,0 +1,140 @@
+
+## What we researched
+
+We decided to try simplifying the design by removing the nested checkboxes, and instead only have a single list of checkboxes, which each still reveal a single ‘Details’ field.
+
+![Screenshot showing a page with the heading ‘Feedback for Peter Essien’. Beneath the heading is a exclaimation mark icon and the bolded text ‘You must not reject candidates for a lack of experience working in a school’. Beaneath this is the subheading ‘Tell Peter Essien why you are rejecting their application’. Beneath this is a set of 9 checkboxes labelled with different reasons. Beneath this is text box with the label ‘Give Peter Essien advice on what to do next. For example, “You should apply again to a different course”’. Beneath this is a green ‘Continue’ button.](updated-rejection-reasons.png "New design for the candidate feedback screen")
+
+We reduced the total number of checkboxes from 28 to 9, by grouping some of the checkboxes that we considered very similar.
+
+The reasons that related to the statutory criteria for initial teacher training (having a degree and GCSE minimum grades, or equivalent) were moved onto separate pages.
+
+The ‘Other’ option was removed – in part so that we could see within the usability research whether participants thought there were any additional options needed.
+
+The content was also edited to have a consistent style and be more descriptive, for example ‘We cannot sponsor a visa’ instead of ‘Visa sponsorship’.
+
+{% set oldList %}
+<h3 class="govuk-heading-s">Qualifications</h3>
+<ul class="govuk-list govuk-list--bullet">
+  <li>No maths GCSE at minimum grade 4 or C, or equivalent</li>
+  <li>No English GCSE at minimum grade 4 or C, or equivalent</li>
+  <li>No science GCSE at minimum grade 4 or C, or equivalent</li>
+  <li>No bachelor’s degree or equivalent</li>
+  <li>Degree does not meet course requirements</li>
+  <li>Could not verify qualifications</li>
+  <li>Other</li>
+</ul>
+
+<h3 class="govuk-heading-s">Personal statement</h3>
+<ul class="govuk-list govuk-list--bullet">
+  <li>Quality of writing</li>
+  <li>Other</li>
+</ul>
+
+<h3 class="govuk-heading-s">Teaching knowledge, ability and interview performance</h3>
+<ul class="govuk-list govuk-list--bullet">
+  <li>Teaching knowledge, ability and interview performance</li>
+  <li>Reasons</li>
+  <li>Subject knowledge</li>
+  <li>Safeguarding knowledge</li>
+  <li>Teaching method knowledge</li>
+  <li>Teaching role knowledge</li>
+  <li>Teaching demonstration</li>
+  <li>Other</li>
+</ul>
+
+<h3 class="govuk-heading-s">Communication, interview attendance and scheduling</h3>
+<ul class="govuk-list govuk-list--bullet">
+  <li>Did not reply to messages</li>
+  <li>Did not attend interview</li>
+  <li>Could not arrange interview</li>
+  <li>Other</li>
+</ul>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>Safeguarding</li>
+  <li>Visa sponsorship</li>
+  <li>Course full</li>
+  <li>Other</li>
+</ul>
+{% endset %}
+
+{% set newList %}
```

Comment:
In the post this reads a bit oddly when its presented as 2 columns as each section doesn't line up. For example everything in the 'After' list sits directly opposite the Qualifications section in the 'Before' list, so it looks like we didn't change anything under personal statement, Teaching knowledge etc. I wonder if instead of having 2 columns we just have the 'Before' and 'After' come one after the other?

<img width="495" alt="Screenshot 2023-03-13 at 14 44 00" src="https://user-images.githubusercontent.com/68232608/224736316-fd543bc1-9dfe-405b-83e6-c4b47bf0d07c.png">